File: library/src/androidTest/java/com/raizlabs/android/dbflow/test/querymodel/TestQueryModel.java
package com.raizlabs.android.dbflow.test.querymodel;

import com.raizlabs.android.dbflow.annotation.Column;
import com.raizlabs.android.dbflow.annotation.QueryModel;
import com.raizlabs.android.dbflow.structure.BaseQueryModel;
import com.raizlabs.android.dbflow.test.TestDatabase;

/**
 * Description:
 */
@QueryModel(databaseName = TestDatabase.NAME)
public class TestQueryModel extends BaseQueryModel {

    @Column
    String newName;

    @Column
    long average_salary;

    @Column
    String department;
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/querymodel/TestQueryModel.kt
package com.raizlabs.android.dbflow.querymodel

import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.QueryModel
import com.raizlabs.android.dbflow.TestDatabase

/**
 * Description:
 */
@QueryModel(database = TestDatabase::class)
data class TestQueryModel(@Column var newName: String? = null,
                          @Column var average_salary: Long = 0,
                          @Column var department: String? = null)
